pages/api/games/[id].js

```diff
@@ -0,0 +1,15 @@
+import dbConnect from "@/db/connect";
+import Games from "@/db/models/Game";
+
+export default async function handler(request, response) {
+  await dbConnect();
+  const { id } = request.query;
+  console.log("🚀  req:", request.query);
+
+  if (request.method === "GET") {
+    const games = await Games.findById(id);
+    response.status(200).json(games);
+  } else {
+    response.status(400).json({ message: "Something went wrong !!!" });
```

Comment:
If you don't plan to support other methods rather than GET, send back a [405](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/405) and the appropriate message.

Reply:
We are planning to implement more methods. Should we leave it or change it to something else?

Reply:
Leave it unless you are covering all possible methods ^^